pasta/augment/inline_test.py

```diff
@@ -0,0 +1,97 @@
+# coding=utf-8
+"""Tests for augment.inline."""
+# Copyright 2017 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ast
+import unittest
+
+from pasta.augment import inline
+from pasta.base import scope
+from pasta.base import test_utils
+
+
+class InlineTest(test_utils.TestCase):
+
+  def test_inline_simple(self):
+    src = 'x = 1\na = x\n'
+    t = ast.parse(src)
+    inline.inline_name(t, 'x')
+    self.checkAstsEqual(t, ast.parse('a = 1\n'))
+
+  def test_inline_multiple_targets(self):
+    src = 'x = y = z = 1\na = x + y\n'
+    t = ast.parse(src)
+    inline.inline_name(t, 'y')
+    self.checkAstsEqual(t, ast.parse('x = z = 1\na = x + 1\n'))
+
+  def test_inline_multiple_reads(self):
+    src = '''\
```

Comment:
I personally prefer using [`textwrap.dedent`](https://docs.python.org/2/library/textwrap.html#textwrap.dedent), but your call.

Reply:
Done-- yeah, I've used this elsewhere, better to be consistent.